docs/content/latest/api/ysql/extensions.md

```diff
@@ -249,14 +295,11 @@ $ ls "$(pg_config --pkglibdir)" | grep <name>
 $ ls "$(pg_config --sharedir)"/extension/ | grep <name>
 ```
 
-Copy those files to the YugabyteDB installation.
-Restart the cluster (or the respective node in a multi-node install).
-Finally, connect to the cluster with `ysqlsh` and run the `CREATE EXTENSION` statement to create the extension.
+Copy the files to the YugabyteDB installation, and restart the cluster (or the respective node in a multi-node install). Finally, connect to the cluster with `ysqlsh` and run the `CREATE EXTENSION` statement to create the extension.
 
-{{< note title="Note" >}}
+{{< note title="Supported extensions" >}}
```

Comment:
I'm not a big fan of notes, especially with our style sheet that highlights them instead of making them less important than the rest of the text. 
We could use "Note that ..." sentences (unless we change the note element style)

Reply:
Yeah, this style does draw too much attention to our very small number of supported extensions :)